arch/arm/src/nrf52/nrf52832_errdata.c

```diff
@@ -154,4 +160,6 @@ void nrf52832_errdata_init(void)
   nrf52832_errdata_16();
```

Comment:
This one only applies to Rev A and causes a far amount of power increase in WFE/WFI

Reply:
The 16 and 66 workarounds already include a check, I would presume it is to detect affected parts